model-optimizer/extensions/front/ThresholdedReluDecomposition.py

```diff
@@ -36,3 +36,4 @@ def find_and_replace_pattern(self, graph: Graph):
             mul.in_port(1).connect(float_greater.out_port(0))
 
             rename_nodes([(node, name + '/TBR'), (mul, name)])
+            node.in_port(0).disconnect()
```

Comment:
Lets just remove the `node` from the graph.

Reply:
Fixed.